docs/operational-playbook.md

```diff
@@ -48,8 +48,8 @@ and add a test for your change before implementing it)
 ### Testing a data or import lambda change
 
 Data is static after being imported. Changes to the data or its
-[import lambdas](../cdk/bin/src/open-data-platform/data-plane/data-import/) require removing
-existing data and re-importing:
+[import lambdas](../cdk/src/open-data-platform/data-plane/data-import/) require removing existing
```

Comment:
nit: one line?

Reply:
Yeah I noticed this points to a `bin` directory that isn't checked into github. So this PR fixes it.